ktor-http/common/src/io/ktor/http/URLUtils.kt

```diff
@@ -129,6 +129,26 @@ public fun Appendable.appendUrlFullPath(
         }
 }
 
+/**
+ * Checks if [Url] has absolute path.
+ */
+public val Url.isAbsolute: Boolean get() = pathSegments.firstOrNull() == ""
```

Comment:
What if you build URL like `Url("/some/path")`? Then `path` is absolute, but url itself is relative. 
Maybe rename this extension to `isAbsolutePath` and introduce `val Url.isAbsolute get() = localhost.isEmpty()`?

Reply:
let's rename it about path